### PR TITLE
Add new TFMs

### PR DESCRIPTION
**Pull request by mairaw**
Should we also add netcoreapp2.0 and netstandard2.0 to this topic?

/cc @richlander @terrajobst

### CODE REVIEW

**Comment by mairaw** [comment]
Also wondering if we should just cut the second paragraph of this topic. I think it just makes the topic more confusing.

**Comment by mairaw** [comment]
@bleroy is that something you could help review?

File: docs/standard/frameworks.md [hunk 1]
```diff
@@ -49,17 +49,17 @@ You can refer to a family of frameworks using long- or short-forms of the framew
...
 | .NET Core Application | 1.1.1          | .NETCoreApp,Version=1.1        | netcoreapp1.1                          | 1.6                   | [Microsoft.NETCore.App](https://www.nuget.org/packages/Microsoft.NETCore.App) |
-| .NET Framework        | 4.6.2          | .NETFramework,Version=4.6.2    | net462                                 | 1.5                   | N/A |
+| .NET Framework        | 4.7            | .NETFramework,Version=4.7      | net47                                  | 1.5                   | N/A |
 
 ## Supported frameworks
 
-A framework is typically referenced by a short target framework moniker or *TFM*. In .NET Standard, this is also generalized to *TxM* to allow a single reference to multiple frameworks. The NuGet clients support the following frameworks. Equivalents are shown within brackets (`[]`).
+A framework is typically referenced by a short TFM. In .NET Standard, this is also generalized to *TxM* to allow a single reference to multiple frameworks. The NuGet clients support the following frameworks. Equivalents are shown within brackets (`[]`).
```
Comment (bleroy) [inline docs/standard/frameworks.md]: The TxM bit, even if not introduced in this change, is confusing. I'm not sur a new acronym adds value.

**Review by bleroy — APPROVED** [review]

File: docs/standard/frameworks.md (hunk 1 above)
Comment (mairaw) [inline docs/standard/frameworks.md]: so you think we should just leave the TFMs listed? I can remove that, no worries. And what about NET Standard 2.0 and .NET Core 2.0. We can add that to the topic already as well, right?

Comment (bleroy) [inline docs/standard/frameworks.md]: That's what I'd do, with maybe a mention that a name in this column may technically cover more than one TFM?

**Comment by mairaw** [comment]
@bleroy is this what you had in mind re. TxMs?

**Pull request merged**